erizo/src/erizo/WebRtcConnection.h

```diff
@@ -82,7 +82,7 @@ class WebRtcConnection: public MediaSink, public MediaSource, public FeedbackSin
    */
   bool setRemoteSdp(const std::string &sdp);
 
-  bool createOffer();
+  bool createOffer(bool videoEnabled, bool audioEnabled, bool bundle);
```

Comment:
I don't want to be too picky here, so this is only informational: the new coding style says we prefer variable names in Snake Case (`video_enabled`, `audio_enabled`, ...). But you can leave them as they're now because we're not changing existing variables yet.